docs/reference/motlp.md

```diff
@@ -173,11 +175,34 @@ When creating a Kubernetes secret, always encode the full string in Base64, incl
 
 ::::::
 
+% ## Rate limiting and throttling
+% 
+% The following rate limits apply to the {{motlp}}:
+% 
+% * n requests per second per API key
+% * m requests per minute per API key
+% 
+% If you exceed these limits, you will receive an HTTP 429 status code.
+
+## Data retention
+
+You can configure the data retention for each data stream from the **Streams** screen. Refer to [Managed data retention](docs-content://solutions/observability/logs/streams/management/retention.md).
```

Comment:
This is applicable only for Serverless afaik. But as we're in MOTel docs, we should be good.

Reply:
@mlunadia I would then propose to leave the draft as is and plan for further docs down the line.